processor/terraform/main.tf

```diff
@@ -0,0 +1,63 @@
+terraform {
+  backend "s3" {
+    encrypt = true
+    key     = "processor.tfstate"
```

Comment:
Since we're going to be targeting services accounts now can we prefix all these with `services/generate/`

Reply:
Moved this to an issue so we can track how state is stored for different accounts.